### PR TITLE
Change version to 2.8 and update build and deployment for LTS

### PR DESCRIPTION
**Pull request by marcschier**
* Moving to Alpine as base image
* Update deploy script to allow deploying from fork
* Adding build scripts to build and deploy using private registry to support quick manual testing
* Update version 2.8.0
* Install IoT Edge LTS (1.1) on Ubuntu 18.04 as part of simulation
* Change from master branch to main branch everywhere

### CODE REVIEW

File: deploy/scripts/edge-setup.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
```
Comment (marcschier) [inline deploy/scripts/edge-setup.sh:1]: Need to undo this, the script should fail at any point  needed.   Or we should add individual return code checks.

File: readme.md [hunk 2]
```diff
@@ -20,46 +18,46 @@ We have also built an application running on Azure that lets you access the serv
...
-- [Read the documentation](https://azure.github.io/Industrial-IoT/).
-- Releases of the platform can be found [here](https://github.com/Azure/Industrial-IoT/releases).
-- [Deploy the pre-built services, the configuration application and edge gateways](docs/deploy/readme.md).
-- Explore other Azure Industrial IoT products using this platform.
-  - [Connected Factory](https://github.com/Azure/Azure-IoT-Connected-Factory) Solution Accelerator which can be deployed from [here](https://www.azureiotsolutions.com/Accelerators).
-  - Also check out our [OPC Vault Application](https://github.com/Azure/azure-iiot-opc-vault-service/tree/master/app).
+To [deploy the Azure Industrial IoT Platform](docs/deploy/readme.md), clone the repository:
+
+  ```bash
+  git clone -b release/2.7.206 https://github.com/Azure/Industrial-IoT
```
Comment (marcschier) [inline readme.md:24]: Must be changed

**Review by mregen — APPROVED** [review]
:shipit:

**Review by cristipogacean — APPROVED** [review]
:shipit:

**Pull request merged**